cypress/tests/core/user-control-panel.js

```diff
@@ -1,30 +1,66 @@
-describe('User Control Panel Test', () => {
-  beforeEach(() => {
-    // given a logged in editor
-    // and a folder that contains a document
-    // and the folder contents view
-    cy.visit('/');
-    cy.autologin();
-    cy.visit('/controlpanel/users');
-    cy.waitForResourceToLoad('@navigation');
-    cy.waitForResourceToLoad('@breadcrumbs');
-    cy.waitForResourceToLoad('@actions');
-    cy.waitForResourceToLoad('@types');
-  });
+if (Cypress.env('API') !== 'guillotina') {
+  describe('User Control Panel Test', () => {
```

Comment:
We removed all the conditionals if Guillotina ... This can be removed as well as it's in cypress/tests/core